src/components/datasets/DatasetOverview.js

```diff
@@ -37,9 +37,9 @@ class DatasetOverview extends Component {
                                value={(new Date(Date.parse(dataset.created))).toLocaleString()} />
                 </Col>
                 <Col span={this.props.isPrivate ? 12 : 8}>
-                    <Spin spinning={this.props.isFetchingTables}>
-                        <Statistic title="Tables"
-                                   value={this.props.isFetchingTables ? EM_DASH : dataset.tables.length} />
+                    <Spin spinning={this.props.isFetchingDatasets}>
+                        <Statistic title="Data types"
+                                   value={this.props.isFetchingDatasets ? EM_DASH : dataset.tables.length} />
```

Comment:
wrong value + if this is kept as a statistic, it should be the number of non-empty data types

Reply:
Done, if you test the behaviour may be odd since I have to better organize the datasets redux stores. Will update tomorrow